js/screens/welcome-screen.js

```diff
@@ -3,3 +3,74 @@ export const welcomeScreen = () => {
    welcomeStepWrapper.innerText = 'Welcome screen';
    return welcomeStepWrapper;
 };
+
+const svgNS = './assets/choose.svg';
+const svg = document.createElementNS(svgNS, 'svg');
+
+function createButton() {
+   const chooseButton = document.createElement('button');
+   const contentButton = document.innerText('Choose');
+   svg.appendChild(chooseButton && contentButton);
+   document.body.appendChild(svg);
+   return chooseButton && contentButton;
+}
+
+/*
+button.addEventListener('click', function (event) {});
+*/
+
+image.onClick = function choose() {
+   image.src = './components/choose.svg';
+   return image.src;
+};
+
+/*
+chooseButton.addEventListener('click', e =>
+{
+   window.location =
+})
+*/
+
+function ultimatePixel() {
+   const h1UltimatePixel = document.createElement('h1');
+   h1UltimatePixel.innerText = 'Ultimate Pixel';
+   document.body.append(h1UltimatePixel);
+   return h1UltimatePixel;
+}
+
+function championships() {
+   const h1Championships = document.createElement('h1');
+   h1Championships.innerText = 'Championships';
+   document.body.append(h1Championships);
+   return h1Championships;
+}
+
+ultimatePixel();
+championships();
+
+function lineBelowUPC(orangeLine) {
+   const orangeLine = document.createElement('div');
+   orangeLine = src('./components/orangeLine.svg');
+   return orangeLine;
+}
+
+function margins(marginHeader, marginFooter) {
+   const marginHeader = document.createElement('div');
+   marginHeader = src('./components/marginHeader.svg');
+   const marginFooter = document.createElement('div');
+   marginFooter = src('./components/marginFooter.svg');
+   return marginNav && marginFooter;
+}
+
+class swords {
+   constructor(leftSword, rightSword) {
+      this.leftSword = left.sword;
```

Comment:
**nitpick:** This is not clear what does this class do - I think it would be good to remove it as long as it does not encapsulate some specific logic. Note: class names should be written with PascalCase in Javascript